forge-gui/res/cardsfolder/upcoming/lock_and_load.txt

```diff
@@ -0,0 +1,7 @@
+Name:Lock and Load
+ManaCost:2 U
+Types:Sorcery
+K:Plot:3 U
+A:SP$ Draw | Defined$ You | NumCards$ X | StackDescription$ SpellDescription | SpellDescription$ Draw a card, then draw a card for each other instant and sorcery spell you’ve cast this turn.
```

Comment:
this is something you need to watch out for, those are two separate draws
they should not be merged
it matters especially for _Alms Collector_

Reply:
Duly noted and edited accordingly.